src/package/name.rs

```diff
@@ -50,6 +50,11 @@ impl PackageName {
         Ok(Self(value))
     }
 
+    /// New package name from an unchecked string.
```

Comment:
Having this means we can violate "type safety". Are there multiple places where we are using this?

I think I only saw one use of it, where we do: `PackageName::unchecked(".")`. If that is the only use, do you think it might be a good strategy to implement `Default` instead and then call `PackageName::default()` there? 👵🏼